packages/server/utils/analytics/analytics.ts

```diff
@@ -369,6 +371,10 @@ class Analytics {
     this.track(userId, 'Summary Email Setting Changed', {subscribeToSummaryEmail})
   }
 
+  teamsLimitEmailSent = (userId: string, orgId: string, type: TeamLimitsEmailType) => {
```

Comment:
@tghanken I added this analytics event so that we track when emails are sent. Would you like this to be edited at all?

Reply:
+1 Could we change this to a generic `Notification Email Sent`? I'd rather have it be usable for other similar types of conversion messaging in the future without changing the name.  We can always expand what we pass into the `type` parameter for other email types.

Reply:
For sure - updated!